src/simulator/git.py

```diff
@@ -0,0 +1,75 @@
+import subprocess
+from simulator.log import error, info
+
+
+def is_inside_git_repo():
+    try:
+        result = subprocess.run(['git', 'rev-parse', '--is-inside-work-tree'],
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                check=True,
+                                text=True)
+        return result.stdout.strip() == 'true'
```

Comment:
Couldn't this use the error level of the process instead, that would seem more reliable to me? I.E. if the output changes, or something else prints out as well your still safe.

I just tested it locally and it seems to return `0` & `128` depending on where it's run from.